simphony/testing/utils.py

```diff
@@ -80,6 +80,18 @@ def create_particles(n=10, restrict=None):
     return particle_list
 
 
+def create_particles_with_id(n=10, restrict=None):
```

Comment:
Good point, please add a separate issue so that we can track the remaining work.